src/main/java/io/github/jhipster/registry/web/rest/JWTToken.java

```diff
@@ -1,24 +0,0 @@
-package io.github.jhipster.registry.web.rest;
-
```

Comment:
@JulienMrgrd please revert this commit alone as this is generated by JHipster so if you wanna change it I suggest you change it in the Generator as well so that its consistent. As i said earlier we need to keep generated code clean as possible so that upgrades are easy